src/cascadia/TerminalControl/SearchBoxControl.h

```diff
@@ -0,0 +1,55 @@
+﻿//
+// SearchBoxControl.h
+// Declaration of the SearchBoxControl class
+//

```

Comment:
Could you add the copyright header we have on all the other files. I think you'd add your name into the header too 😊

Reply:
added